sk0-front/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  baseUrl: ''
+  baseUrl: "http://localhost:3000",
```

Comment:
It is a deployable build environment. Shouldn't we use ```process.env ||  for the baseurl```?

Reply:
this part of the angular is less dynamic as the [vite](https://vitejs.dev/) `import.meta.ENV`, but it might be available soon if angular is moving towards esbuild later 

with vitejs build it is pretty simple to template envvars https://vitejs.dev/guide/env-and-mode

Reply:
I will try one more thing :)

Reply:
nope, their new esbuild version is not working neither 👎

Reply:
yeah @rolcho this is messed up :D I do not even know now how we did it :D

Reply:
so we should use like we did in our previous project for different build cases?

environment.production.ts:
```typescript
export const environment = {
  production: true,
  SERVER_URL: 'https://XXXXXXXXXX.up.railway.app',
};
```

environment.local.ts:
```typescript
export const environment = {
  production: true,
  SERVER_URL: 'http://localhost:3000,
};
```

angular.json: 
```typescript
// ...
           "local": {
              "buildOptimizer": false,
              "optimization": false,
              "vendorChunk": true,
              "extractLicenses": false,
              "sourceMap": true,
              "namedChunks": true,
              "fileReplacements": [
                {
                  "replace": "src/environments/environment.ts",
                  "with": "src/environments/environment.local.ts"
                }
              ]
            }
          },
          "defaultConfiguration": "local"
```
?

Reply:
yes, yes angular is not flexible here
(Which is a funny sentence)